astropy/units/__init__.py

```diff
@@ -16,6 +16,8 @@
 # this also makes it easier to understand where most time is spent
 # (e.g., using python -X importtime).
 
+from .core import set_enabled_units
+
 # isort: off
 from .core import *
 from .quantity import *
```

Comment:
Do we want to keep this line as it is?

Reply:
``.core`` is imported first, so it should be fine. It would eventually be nice to properly isort this module, but that's a task for another time :).